policyengine_canada/parameters/gov/provinces/sk/tax/income/credits/sk_caregiver_amount/age_threshold/infirm.yaml

```
@@ -0,0 +1,17 @@
+description: Saskatchewan provides the Caregiver Tax Credit Amount for filer with infirm dependants at or above this age threshold.
```

Based on the Python file it's younger not older
```suggestion
description: Saskatchewan provides the Caregiver Tax Credit Amount for filer with infirm dependants this age or younger.
```